vertx-web/src/main/java/io/vertx/ext/web/Route.java

```diff
@@ -168,7 +168,7 @@
 
   /**
    * If true then the normalised request path will be used when routing (e.g. removing duplicate /)
-   * Default is true
+   * Defaults to useNormalisedPathByDefault in the router
```

Comment:
You should make a reference to the Router#useNormalisedPath class so readers can follow